### PR TITLE
acpi: add Indonesian translation

### PR DESCRIPTION
**Pull request by punkestu**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.id/linux/acpi.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Tampilkan informasi suhu dalam Fahrenheit:
+
+`acpi -tf`
+
+- Tampilkan semua informasi:
+
+`acpi -V`
+
+- Esktrak informasi dari `/proc` daripada `/sys`:
```
Comment (reinhart1010) [inline pages.id/linux/acpi.md:26]:
```suggestion
- Ekstrak informasi dari `/proc` daripada `/sys`:
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Thanks for your contribution. Here are some of my notes to review.

**Review by reinhart1010 — APPROVED** [review]
Thanks

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**